libs/sdk-react-native/src/index.tsx

```diff
@@ -568,6 +592,21 @@ export const refund = async (swapAddress: string, toAddress: string, satPerVbyte
     return response
 }
 
+export const fetchReverseSwapFees = async (): Promise<ReverseSwapPairInfo> => {
```

Comment:
@ok300  These methods should be implemented in BreezSDK (android & iOS modules).

Reply:
Done, although by hand + not sure how to validate the syntax.

Are there any CLI commands I could run to compile or check the syntax of the react native modules we have here?